src/view/channel-detail.js

```diff
@@ -54,9 +54,14 @@ const ChannelDetailView = ({ store, nav }) => (
         <DetailField name="Balance">
           {store.selectedChannel.localBalanceLabel} {store.unitLabel}
         </DetailField>
-        <Button style={styles.deleteBtn} onPress={() => nav.goChannelDelete()}>
-          <ButtonText style={styles.deleteBtnText}>CLOSE CHANNEL</ButtonText>
-        </Button>
+        {store.selectedChannel.statusLabel === 'Closed' ? null : (
```

Comment:
We also don't want to display the button when the channel is `pending-force-closing` or `pending-closing`, so a better check might be:
```
{!store.selectedChannel.closingTxId ? null : ( ... ) }
```

Reply:
closingTxId doen't work. See result.
![screen shot 2019-02-27 at 3 26 54 pm](https://user-images.githubusercontent.com/2033945/53532630-71d3c280-3aac-11e9-8eb6-36e0ddaa7385.png)
![screen shot 2019-02-27 at 2 22 51 pm](https://user-images.githubusercontent.com/2033945/53532632-7304ef80-3aac-11e9-81c9-3a4f6c106066.png)
![screen shot 2019-02-27 at 2 22 43 pm](https://user-images.githubusercontent.com/2033945/53532633-739d8600-3aac-11e9-975f-5c4015bb3984.png)
![screen shot 2019-02-27 at 2 22 36 pm](https://user-images.githubusercontent.com/2033945/53532634-74361c80-3aac-11e9-8e90-e0c610c0cd0c.png)

Reply:
Instead just simply matching for the keyword 'close' or 'closing' works reliably.

Reply:
``` {/Close|Closing/i.test(store.selectedChannel.statusLabel) ? null : (...)}```